### DISCUSSION TITLE
Default config file

### DISCUSSION THREAD

**Discussion by JulianLundh** [philippe44/SpotConnect#40] [Ideas]
I would love to have a default config file, so I know what option and the syntax. Makes it much easier that to guess. Love your tool btw!

**Comment by philippe44** [comment]
When you use -i [file] that's not enough?

**Comment by JulianLundh** [comment]
im not really following you how to use -1? 

i also have a blank "config.xml" in the same folder but it still says it can't find any

![bild](https://github.com/philippe44/SpotConnect/assets/108955495/535f423d-61d1-45cf-b8ae-1e859fea4466)

**Reply by philippe44** [reply]
-i (not 1) creates a config file with options set to default. Now, if no player is detected, creating a file won't help. Players are not added *because* they are listed in the config file, the config file only tells spotconnect what parameters to use *after* it discovered a player.
